test/CloudNative.CloudEvents.UnitTests/AspNetCore/HttpResponseExtensionsTest.cs

```diff
@@ -114,11 +114,17 @@ public async Task CopyToHttpResponseAsync_Batch()
 
             var content = GetContent(response);
             Assert.Equal(MimeUtilities.BatchMediaType + "+json; charset=utf-8", response.ContentType);
+            Assert.NotNull(response.ContentType);
             var parsedBatch = new JsonEventFormatter().DecodeBatchModeMessage(content, new ContentType(response.ContentType), extensionAttributes: null);
             AssertBatchesEqual(batch, parsedBatch);
         }
 
-        private static HttpResponse CreateResponse() => new DefaultHttpResponse(new DefaultHttpContext()) { Body = new MemoryStream() };
+        private static HttpResponse CreateResponse()
+        {
+            var response = new DefaultHttpContext().Response;
+            response.Body = new MemoryStream();
+            return response;
+        }
```

Comment:
Nit: please add a lack line after this method.

Reply:
Will do